device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-C36/1/j2p-a7800r3a-36d-36x400G.config.bcm

```diff
@@ -0,0 +1,984 @@
+soc_family=BCM8885X
+
```

Comment:
`system_ref_core_clock_khz=1600000` is missing ?

Reply:
This is default for j2c+, we do not need to specify it explicitly in bcm conf file.